### ISSUE TITLE
[CLI] Fix `spawn` error when initializing a new xata database

### ISSUE THREAD

**Issue by creative-tutorials** [xataio/client-ts#1452]
**Describe the bug**

When I try to initialize xata into my new project, I get an error that says:

```shell
Running pnpm add @xata.io/client
    Error: spawn EINVAL
    Code: EINVAL
```

It doesn't even create the `xata.ts` file or even install the `@xata.io/client` package.

**To Reproduce**

### Command I used

```shell
xata init
```

### Output of the error

```shell
Running pnpm add @xata.io/client
    Error: spawn EINVAL
    Code: EINVAL
```

**Expected behavior**

Normally when running `xata init` it generates the `.xata.ts` file and installs the `@xata.io/client` package

**Software version**

The CLI version I'm running is: `@xata.io/cli/0.15.11 win32-x64 node-v20.12.2`

**Additional context**

**Comment by SferaDev** [comment]
Fixed by #1466

**Comment by creative-tutorials** [comment]
> Fixed by #1466 

You're amazing
